packages/salesforcedx-vscode-core/src/commands/auth/orgLoginWeb.ts

```diff
@@ -129,6 +134,10 @@ export class OrgLoginWebContainerExecutor extends SfCommandletExecutor<AuthParam
 class OrgLoginWebExecutor extends SfCommandletExecutor<AuthParams> {
   protected showChannelOutput = false;
 
+  constructor(extensionContext?: vscode.ExtensionContext) {
```

Comment:
this is going to get flagged by that eslint rule for no-useless-constructor.  SfCommandletExecutor doesn't have a constructor so this does nothing